src/mapOfPeople.js

```diff
@@ -40,7 +40,12 @@
  * @return {Map}
  */
 function mapOfPeople(people) {
-  // write code here
+  const peopleInRoom = new Map();
+  people.forEach((men, time) => {
+    !peopleInRoom.has(men)
```

Comment:
Ok, but simpler and more readable to use condition without `!` and swap expressions